packages/core/src/util/renderer.ts

```diff
@@ -139,6 +140,7 @@ export const mapStateToControlProps = (state, ownProps): StatePropsOfControl =>
       controlElement.scope !== undefined && isRequired(ownProps.schema, controlElement.scope);
   const resolvedSchema = Resolve.schema(ownProps.schema, controlElement.scope);
   const description = resolvedSchema !== undefined ? resolvedSchema.description : '';
+  const config = getConfig(state);
```

Comment:
didn't we want to merge the config of the state with the options of the uischema?